README.md

```diff
@@ -1,29 +1,42 @@
 # strong-build
 
-Build a node application package, preparing it for production.
+Build a node application package, preparing it for deploy to production.
 
-## Packaging
 
-Best practice when preparing a node (or any) application for production is to
-bundle its dependencies so that there are no deploy-time dependencies on
-external services.  Not doing so for a node applications leads to deploy time
-dependencies on external services such as npmjs.org, with possibly catastrophic
-results.
+## Install
+
+`sl-build` is made available through the
```

Comment:
This is the first mention of `sl-build`. Might want to reword this section to state that `sl-build` is the command that this module installs.